kuma/sample_apps.go

```diff
@@ -21,3 +25,35 @@ func (kuma *Kuma) installSampleApp(del bool, namespace string, templates []adapt
 
 	return status.Installed, nil
 }
+
+func (kuma *Kuma) sidecarInjection(namespace string, del bool) error {
+	kclient := kuma.KubeClient
+	if kclient == nil {
+		return ErrNilClient
+	}
+
+	// updating the label on the namespace
+	ns, err := kclient.CoreV1().Namespaces().Get(context.TODO(), namespace, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	// updating the annotations on the namespace
+	if ns.ObjectMeta.Annotations == nil {
+		ns.ObjectMeta.Annotations = map[string]string{}
+	}
+	ns.ObjectMeta.Annotations["kuma.io/sidecar-injection"] = "enabled"
+
+	if del {
+		delete(ns.ObjectMeta.Annotations, "kuma.io/sidecar-injection")
+	}
+
+	fmt.Println(ns.ObjectMeta)
```

Comment:
I think we can remove this log?

Reply:
Yeah sure I am removing this `fmt.Println(ns.ObjectMeta)`